src/lib/receptions/SequencescapeKinnexTubes.js

```diff
@@ -40,7 +40,7 @@ const fetchLabwareForReception = async ({ requests, barcodes, requestOptions })
  *
  * @returns {Array} Array of attribute keys
  */
-const getAttributeKeys = () => ['tubes_attributes']
+const getAttributeKeys = () => ['compound_sample_tubes_attributes']
```

Comment:
Where is this function used, can't see where its used from a brief look at the code base, same goes for the other receptions with this function.

Reply:
This function is set up in the Receptions config and is accessed dynamically depending on which reception type is being used which helps component to figure out which attribute keys to use for each workflow, allowing correct attributes to be retrieved for each reception type.